src/plans/stepsize.jl

```diff
@@ -756,6 +757,64 @@ function show(io::IO, a::NonmonotoneLinesearch)
 end
 get_message(a::NonmonotoneLinesearch) = a.message
 
+@doc raw"""
+    PolyakStepsize <: Linesearch
+
+
+Compute a step size
+
+````math
+α_i = \frac{f(p^{(k)}) - f_{\text{best}}+γ_k}{\lVert{ ∂f(p^{(k)})} \rVert^2},
```

Comment:
We should consistently use either `i` or `k` for the current iteration.

Reply:
I somehow started the package with `i` but otherwise usually use `k`, so I sometimes confuse them.